src/webapp/data/SampleProgram.js

```diff
@@ -25,14 +25,14 @@ result: .word32 0
 	; ---
 
 	.text
-	
+
```

Comment:
(ah, the white space trimmer in vim, I forgot to disable it -.-)

Reply:
No worries, that's ok :)